app/controllers/posts_controller.rb

```diff
@@ -653,6 +653,11 @@ def create_params
       result[:is_warning] = false
     end
 
+    if params[:no_bump] == "true"
+      raise Discourse::InvalidParameters.new(:no_bump) unless current_user.admin?
```

Comment:
oh ... this is a staff feature, not only an admin feature, also please move to guardian